content/docs/concepts/components/lifecycle/analyze.md

```diff
@@ -1,11 +1,16 @@
 +++
 title="Analyze"
-weight=2
+weight=1
 summary="Restores files that buildpacks may use to optimize the build and export phases."
 +++
 
 {{< param "summary" >}}
 
+Prior to `Platform API 0.7`, the `analyzer` was responsible for analyzing the metadata from the cache and the previously built image (if available) to determine what layers can or cannot be reused.
+This information is used during the `export` phase in order to avoid re-uploading unchanged layers.\
+Starting from `Platform API 0.7`, the `analyze` phase runs before the `detect` phase in order to validate registry access for all images that are used during the build as early as possible. This validation now happens in the first lifecycle phase, as part of the `analyzer`, before running both the `detector` and the `builder`. In this way it provides faster failures for end users. The other responsibilities of the `analyzer` were moved to the `restorer`.\
```

Comment:
```suggestion
Starting from `Platform API 0.7`, the `analyze` phase runs before the `detect` phase in order to validate registry access for all images that are used during the build as early as possible. In this way it provides faster failures for end users. The other responsibilities of the `analyzer` were moved to the `restorer`.\
```
(analyzer always ran before builder)

Reply:
I know that the analyzer always ran before the builder, just thought that it's worth emphasizing it, but I don't have strong opinion about this so I'll make this change.
Thanks.

Reply:
Please see https://github.com/buildpacks/docs/pull/412/commits/3fae08256362ad2fc1293dfba4a5543d1272e2fa.